superbench/cli/_handler.py

```diff
@@ -21,49 +21,104 @@ def check_argument_file(name, file):
         name (str): argument name.
         file (str): file path.
 
+    Returns:
+        str: Absolute file path if it exists.
+
     Raises:
         CLIError: If file does not exist.
     """
-    if file and not Path(file).exists():
-        raise CLIError('{} {} does not exist.'.format(name, file))
+    if file:
+        if not Path(file).exists():
+            raise CLIError('{} {} does not exist.'.format(name, file))
+        return str(Path(file).resolve())
+    return file
 
 
-def version_command_handler():
-    """Print the current SuperBench tool version.
+def split_docker_domain(name):
+    """Split Docker image name to domain and remainder part.
+
+    Ported from https://github.com/distribution/distribution/blob/v2.7.1/reference/normalize.go#L62-L76.
+
+    Args:
+        name (str): Docker image name.
 
     Returns:
-        str: current SuperBench tool version.
+        str: Docker registry domain.
+        str: Remainder part.
     """
-    return superbench.__version__
+    i = name.index('/')
+    domain, remainder = name[:i], name[i + 1:]
+    default_domain, legacy_domain = 'docker.io', 'index.docker.io'
+    if i != -1 or (':' not in domain and '.' not in domain and domain != 'localhost'):
+        domain, remainder = default_domain, name
+    if domain == legacy_domain:
+        domain = default_domain
+    if domain == default_domain and '/' not in remainder:
+        remainder = 'library/{}'.format(remainder)
+    return domain, remainder
```

Comment:
Why do we need to return remainder? Only one function uses it and the remainder will be dropped. Why not implement it as get_docker_domain()?